assets/js/main.js

```diff
@@ -1,40 +1,64 @@
+/* eslint-disable max-classes-per-file */
+
+// Class constructor
```

Comment:
[Optional]
You can use multiple files and [import your classes](https://www.digitalocean.com/community/tutorials/js-modules-es6) to avoid this error. Suppressing linter errors should be the last option to choose.

Reply:
done!